python/cudf_kafka/setup.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+import os
+import shutil
+import sysconfig
+from distutils.sysconfig import get_python_lib
+
+import numpy as np
+from Cython.Build import cythonize
+from setuptools import find_packages, setup
+from setuptools.extension import Extension
+
+import versioneer
+
+install_requires = ["cudf", "cython"]
+
+cython_files = ["cudf_kafka/_lib/*.pyx"]
+
+CUDA_HOME = os.environ.get("CUDA_HOME", False)
+if not CUDA_HOME:
+    path_to_cuda_gdb = shutil.which("cuda-gdb")
+    if path_to_cuda_gdb is None:
+        raise OSError(
+            "Could not locate CUDA. "
+            "Please set the environment variable "
+            "CUDA_HOME to the path to the CUDA installation "
+            "and try again."
+        )
+    CUDA_HOME = os.path.dirname(os.path.dirname(path_to_cuda_gdb))
+
+if not os.path.isdir(CUDA_HOME):
+    raise OSError(f"Invalid CUDA_HOME: directory does not exist: {CUDA_HOME}")
+
+cuda_include_dir = os.path.join(CUDA_HOME, "include")
+
+CUDF_ROOT = os.environ.get("CUDF_ROOT", "../../cpp/build/")
+
+try:
+    nthreads = int(os.environ.get("PARALLEL_LEVEL", "0") or "0")
+except Exception:
+    nthreads = 0
+
+extensions = [
+    Extension(
+        "*",
+        sources=cython_files,
+        include_dirs=[
+            "../../cpp/include/cudf",
+            "../../cpp/include",
+            "../../cpp/libcudf_kafka/include/cudf_kafka",
+            os.path.join(CUDF_ROOT, "include"),
+            os.path.join(CUDF_ROOT, "_deps/libcudacxx-src/include"),
+            os.path.join(
+                os.path.dirname(sysconfig.get_path("include")),
+                "libcudf/libcudacxx",
+            ),
+            os.path.dirname(sysconfig.get_path("include")),
+            np.get_include(),
+            cuda_include_dir,
+        ],
+        library_dirs=(
+            [get_python_lib(), os.path.join(os.sys.prefix, "lib")]
+            + [
+                os.path.join(
+                    os.path.join(os.sys.prefix, "libcudf_kafka"), "lib"
+                )
```

Comment:
Wouldn't the `libcudf_kafka.so` just be installed directly into `lib` as opposed to in a separate folder?

Reply:
yes, that was something I copied from the previous structure and forgot to move. Removing